src/Metable.php

```diff
@@ -75,6 +75,20 @@ public function setMeta(string $key, $value): void
         }
     }
 
+    /**
+     * Add or update many `Meta` values.
+     *
+     * @param array|Traversable $array
+     *
+     * @return void
+     */
+    public function setManyMeta($array): void
```

Comment:
Can't decide if I like the naming 😆 . What you put is certainly workable, but I'm also questioning other options. `bulkSetMeta`? `setMultipleMeta`? 

Thoughts?

Reply:
I used `Many` because of the many (har har) similar usages in Laravel itself. For example:
- `Cache` has `many` and `putMany()`
- Relationships have `saveMany()`
- `Config` has `getMany()`
- Models have `findMany()`

Interestingly, `Cache` has `deleteMultiple()` instead of `deleteMany()`, as well as `getMultiple()` (which is basically `many()` with a possible default value instead of `null`) and `setMultiple()` (which just wraps `putMany()`). The `Multiple` methods seem to exist so that `Cache` is PSR-6 compliant.

I'd still vote for `Many` over `Multiple` though 😉